dubbo-common/src/main/java/org/apache/dubbo/common/constants/CommonConstants.java

```diff
@@ -643,17 +595,130 @@ public interface CommonConstants {
 
     String REST_SERVICE_DEPLOYER_URL_ATTRIBUTE_KEY = "restServiceDeployerAttributeKey";
 
-    String SERVICE_DEPLOYER_ATTRIBUTE_KEY = "serviceDeployer";
-
-    String DUBBO_MANUAL_REGISTER_KEY = "dubbo.application.manual-register";
-
-    String DUBBO2_COMPACT_ENABLE = "dubbo.compact.enable";
-
     String POD_NAMESPACE = "POD_NAMESPACE";
 
     String CLUSTER_DOMAIN = "CLUSTER_DOMAIN";
 
     String EXT_PROTOCOL = "ext.protocol";
 
     String IS_EXTRA = "isExtra";
+
+    interface SystemProperty {
```

Comment:
Add some javadoc to to describe the purpose.

Reply:
done.